### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by notReally08**

### CODE REVIEW

File: custom-cakes/index.html [hunk 1]
```diff
@@ -4,10 +4,70 @@
...
+  <link rel="stylesheet" href="./modules/footer/__navigation/footer__navigation.css">
+  <link rel="stylesheet" href="./modules/footer/__social-list/footer__social-list.css">
+  <link rel="stylesheet" href="./modules/footer/__social-item/footer__social-item.css">
+  <link rel="stylesheet" href="./modules/footer/__social-item/footer__social-link.css">
   <title>Cakes</title>
 </head>
 <body>
-  <h1>Hello world!</h1>
+  <footer class="footer">
+    <div class="container footer__container">
```
Comment (AntonPashkowskiy) [inline custom-cakes/index.html]: the footer should be placed inside the footer container not outside

File: custom-cakes/modules/footer/__container/footer__container.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+.footer__container {
+  position: relative;
+}
+
+.footer__container::after {
```
Comment (AntonPashkowskiy) [inline custom-cakes/modules/footer/__container/footer__container.css]: What is it?

Comment (AntonPashkowskiy) [reply custom-cakes/modules/footer/__container/footer__container.css]: Oh. It's a separator line. Could we replace it with just a border-bottom of the previous container? It'll be more obvious

**Review by AntonPashkowskiy — CHANGES_REQUESTED** [review]

File: custom-cakes/index.html (hunk 1 above)
Comment (notReally08) [reply custom-cakes/index.html]: Fixed.
I supposed .footer__container to be .inner container. So I added additional inner section named "footer__top" to be the container I originally wanted to be.

File: custom-cakes/modules/footer/__container/footer__container.css (hunk 2 above)
Comment (notReally08) [reply custom-cakes/modules/footer/__container/footer__container.css]: Fixed.
I added aditional container (footer__bottom) for the social icons and copyright block. Also I duplicate the social list to the footer__top section to show it in the mobile and desktop versions.
In the footer__bottom I use the same social list for the tablet version.
Used media queries and display: none/block(or flex)

**Review by AntonPashkowskiy — APPROVED** [review]

**Pull request merged**